rs/embedders/benches/heap.rs

```diff
@@ -134,23 +134,27 @@ fn loop_body(op: &str, dir: Dir, initial_offset: usize, size: Size, step: Step)
             )
         }
         Dir::Bwd => {
+            let end = size as usize - 1 - offset;
             format!(
                 r#"
                 (local.set $address (i64.const {end}))
                 (loop $loop
                     {op}
                     (local.set $address (i64.sub (local.get $address) (i64.const {step})))
-                    (br_if $loop (i64.ge_s (local.get $address) (i64.const {initial_offset})))
+                    (br_if $loop (i64.ge_s (local.get $address) (i64.const 0)))
```

Comment:
The forward direction starts at offset, while the backward direction goes down to zero - is that intended?

Reply:
The offset is to start from the next page, not to limit the beginning of the working set. For example, for offset=1 page, size=6 pages (range [0..5]), step=2 pages (similar to what we use in the `setup` function) for forward direction it should be 1, 3, 5, and for backward direction it should be 4, 2, 0. Does this make sense?

Reply:
Got it now! Could you please add a comment to specify the requirements/expectations on offset?